internal/handler/Handler.go

```diff
@@ -66,36 +67,58 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		h.updateEmployee(w, r)
 		return
 	default:
-		fmt.Println("Error: Not Found!")
+		fmt.Println("error: not found")
 		http.NotFound(w, r)
 		return
 	}
 }
 
 func (h *Handler) getPositions(w http.ResponseWriter, r *http.Request) {
-	positions:=h.service.GetPositions()
+	query, err := url.ParseQuery(r.URL.RawQuery)
+	limit := query["limit"][0]
+	offset := query["offset"][0]
```

Comment:
if you send request without `limit` or  `offset` parameter that code will lead to panic.

Reply:
better to convert these params to int on the handler level and then use them without additional conversion.

Reply:
I thought about it, okay i do it.

Reply:
+